.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
```

Comment:
I don't think we have this in the main Zig repository. Why do we need it here but not there?

Reply:
https://github.com/ziglang/zig/blob/master/.gitattributes#L1-L3 the main repo limits it to zig files
i made this pr before that code was there and i normalize LF on all my files so thats why i used `*`